tiup/tiup-cluster-no-sudo-mode.md

```diff
@@ -90,6 +105,10 @@ This document describes how to use the TiUP no-sudo mode to deploy a cluster.
    
 ## Manually repair failed check items
 
+> **Note:**
+>
+> If you use a minimal install, make sure the `tar` package is installed. Otherwise the `tiup cluster check` command will fail.
```

Comment:
What does "minimal install" mean? Is there any reference link?

Reply:
https://docs.rockylinux.org/guides/installation/#software-selection

This is in the software selection during installation

Reply:
https://docs.rockylinux.org/guides/installation/#software-selection

This is in the software selection during installation 

A screenshot that I found with a Google search:
![9-RockyLinux9-software-selection.jpg](https://github.com/user-attachments/assets/28ec3254-038d-4276-9726-42e9495e8127)